src/main/java/com/simibubi/create/events/CommonEvents.java

```diff
@@ -252,6 +253,7 @@ public static void register() {
 		UseBlockCallback.EVENT.register(FluidBottleItemHook::preventWaterBottlesFromCreatesFluids);
 		ProjectileImpactCallback.EVENT.register(BlazeBurnerHandler::onThrowableImpact);
 		EntityReadExtraDataCallback.EVENT.register(ExtendoGripItem::addReachToJoiningPlayersHoldingExtendo);
+		UseBlockCallback.EVENT.register(FilteringHandler::onBlockActivated);
```

Comment:
what do these changes solve? I haven't noticed any issues with filters.

Reply:
this change only takes effect into a dedicated server (not a LAN world / integrated server)
when the use event being set only on client, the server doesnt know that you set up an filter, resulting in no changes at all